### PR TITLE
feat(client-go): added ps endpoint

### PR DESCRIPTION
**Pull request by Joshua-Anderson**

### CODE REVIEW

File: client-go/cmd/utils.go [hunk 1]
```diff
@@ -66,3 +67,13 @@ func load(appID string) (*client.Client, string, error) {
 
 	return c, appID, nil
 }
+
+func drinkOfChoice() string {
```
Comment (mboersma) [inline client-go/cmd/utils.go]: And to think our BDFL once wanted to get rid of this "unprofessional" message he himself wrote! Thanks to @bacongobbler and you, it lives on. See #2393 for some laughs. :coffee:

File: client-go/controller/models/ps/ps.go [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+	if err != nil {
+		return []api.Process{}, err
+	}
+
+	if status != 200 {
+		return []api.Process{}, errors.New(body)
+	}
+
+	processes := api.Processes{}
```
Comment (mboersma) [inline client-go/controller/models/ps/ps.go]: Maybe the local variable should be `procs`? I had to read the `processes.Processes` line a couple times before it parsed correctly.

File: client-go/controller/models/ps/ps_test.go [hunk 3]
```diff
@@ -0,0 +1,287 @@
...
+			res.Write(nil)
+			return
+		}
+
+		res.WriteHeader(http.StatusNoContent)
+		res.Write(nil)
+		return
+	}
+
+	fmt.Printf("Unrecongized URL %s\n", req.URL)
```
Comment (mboersma) [inline client-go/controller/models/ps/ps_test.go]: Typo: "Unrecognized"

**Comment by mboersma** [comment]
Code LGTM once the minor typo in the tests is fixed.

**Comment by Joshua-Anderson** [comment]
@mboersma Fixed!

**Comment by technosophos** [comment]
Code LGTM

**Pull request merged**